plugins/plugin.proto

```diff
@@ -84,18 +96,29 @@ message PluginResponse {
   // status code.
   repeated string error = 1;
   
-  // text output
+  // text output, typically written to stdout by openapic.
```

Comment:
>typically written to stdout by openapic.

Is this mainly for debugging ? Is the writing of Text on stdout mainly for something like chaining plugins ?